docs/securing_fledge.rst

```diff
@@ -222,6 +222,25 @@ Fledge provides a mechanism to limit the age of passwords in use within the syst
 
 Whenever a user logs into Fledge the age of their password is checked against the maximum allowed password age. If their password has reached that age then the user is not logged in, but is instead forced to enter a new password. They must then login with that new password. In addition the system maintains a history of the last three passwords the user has used and prevents them being reused.
 
+Management Script Login
+-----------------------
+
+Fledge supports a number of mechanisms for user authentication when using the *fledge* script to manage startup, shutdown and other operations.
+
+  - The default authentication mechanism will prompt the entry of user name and password interactively.
+
+  - The user may set one or both of the FLEDGE_USER and FLEDGE_PASSWORD environment variables to automate the entry of the username or password.
+
+  - The user name pass either or both of the *-u* and *-p* flags to set the username and password on the command line.
+
+  - The user name pass the *-c* flag and a path to a certificate file that is used to authenticate the user.
+
+  - The user creates an authentication file that contains the username and password
+
+.. note::
+
+   In all the the cases where a password is stored or passed to the script this is done in plain text. Care should be taken to protect these plain text password. In the case of the authentication file, the file permissions must be such that only the Linux user is able to read the file.
```

Comment:
Except for certificate-based authentication, in all other cases where a password is stored or passed to the script, it is done in plain text. Care must be taken to protect these plain text passwords. For authentication files, the file permissions should be configured so that only the Linux user has the ability to read the file.